src/napari_epitools/_widget.py

```diff
@@ -14,14 +15,15 @@
 from magicgui.widgets.bases import Widget
 from napari import current_viewer
 from napari.qt.threading import thread_worker
-from napari.utils.notifications import show_error
 
 from napari_epitools.analysis import (
     calculate_cell_statistics,
     calculate_projection,
     calculate_segmentation,
 )
 
+logger = logging.getLogger(__name__)
```

Comment:
Why not import like
```python
import napari-epitools
napari-epitools.logger
```

Reply:
doing `logger = logging.getLogger(__name__)` is creating a new logger for this module rather than using the one defined in `napari_epitools.__init__.py`. By calling this logger `__name__`, the logging library will look at the package structure to create a hierarchy of loggers based on the structure. When calling e.g. `logger.info()` or `logger.log()`, the logging library will move up the hierarchy and log events using the handlers defined throughout the hierarchy. So in our case it will use the handler defined in `napari_epitools.__init__.py` to log the event